lib/streamlit/hello/demos.py

```diff
@@ -45,86 +45,6 @@ def intro():
     )
 
 
-# Turn off black formatting for this function to present the user with more
-# compact code.
-# fmt: off
-def mapping_demo():
-    import streamlit as st
-    import pandas as pd
-    import pydeck as pdk
-
-    from urllib.error import URLError
```

Comment:
The PR comments make it sound like the mapping demo should still be present, but we show an error message if PyDeck isn't installed, but the code looks like the demo is gone - what's the intended behavior?